sorc/ncep_post.fd/INITPOST_MPAS.F

```diff
@@ -1761,7 +1687,6 @@ SUBROUTINE INITPOST_MPAS
         IM,1,JM,1,IM,JS,JE,1)
        do j = jsta_2l, jend_2u
         do i = 1, im
-          MRSHLTR ( i, j ) = dummy (i, j )  ! Shelter Mixing ratio
```

Comment:
@jaymes-kenyon I see the values of 'dummy' is also assigned to 'qv2m'. The variable 'qv2m' is written out as grib2 data via ID 760 in SURFCE.f. Thereforce,  I think IDs 414 and 760 serve duplicated functions in UPP. This might be a good time to clean up all ID 414 (MRSHLTR) related configurations, which are include in ALLOCATE_ALL.f, DEALLOCATE.f, INITPOST.F, MPI_FIRST.f.